recipe/meta.yaml

```diff
@@ -1,14 +1,21 @@
-{% set version = "1.12.0" %}
+{% set version = "1.13.0" %}
 
 package:
   name: scipy
   version: {{ version }}
 
 source:
   - url: https://github.com/scipy/scipy/releases/download/v{{ version }}/scipy-{{ version }}.tar.gz
-    sha256: 4bf5abab8a36d20193c698b0f1fc282c1d083c94723902c447e5d2f1780936a3
+    sha256: 58569af537ea29d3f78e5abd18398459f195546bb3be23d16677fb26616cc11e
     patches:                                                     # [s390x]
       - patches/0002-arff-nodata-test-remove-endian-check.patch  # [s390x]
+  # The  version of pocketfft pinned to scipy 1.13.0 is incompatible with osx <10.15.
+  # This is fixed in pocketfft commit https://github.com/mreineck/pocketfft/commit/33ae5dc94c9cdc7f1c78346504a85de87cadaa12
+  # See: https://github.com/scipy/scipy/issues/20300
+  # TODO remove on the next update
+  - git_url: https://github.com/scipy/pocketfft.git              # [osx and x86_64]
+    git_rev: 9367142748fcc9696a1c9e5a99b76ed9897c9daa            # [osx and x86_64]
+    folder: pocketfft                                            # [osx and x86_64]
```

Comment:
Is this the same as creating a patch from a commit? If not, what's the difference and why is this method preferred?

Reply:
pocketfft is a submodule in the scipy project. A commit patch wouldn't help us here, as it only brings the subproject pin. See https://patch-diff.githubusercontent.com/raw/scipy/scipy/pull/20333.patch